INSTALL.md

```diff
@@ -38,15 +38,15 @@ conda install -c conda-forge onnx
 
 ## Build ONNX from Source
 
-Before building from source uninstall any existing versions of ONNX `pip uninstall onnx`.
+Before building from source uninstall any existing versions of ONNX via ``pip uninstall onnx``.
```

Comment:
Single backtick should be good for markdown ?

Reply:
As far as I can remember, single backtick is not an operator in Markdown?

Reply:
Two backticks are used for escaping: https://www.markdownguide.org/basic-syntax/#code

in rst we need two backticks for code blocks

Reply:
Thank you, I revert it.